vmck/migrations/0005_auto_20190812_1258.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.4 on 2019-08-12 12:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('vmck', '0004_job_name'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='job',
+            name='name',
+            field=models.CharField(default='default', max_length=32),
```

Comment:
You can collapse the migrations. Easiest way is to roll them back locally (`./manage.py migrate vmck 0003`), remove the migration files, and generate them again.